Bottom: b0d26700067fa09254ee8df365690f91fd2ac5fb
Top:    d90f6a06ceeb5ffd6ef8cc2c97fe0494dcd7f238
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-07-12 09:24:59 -0700

Refresh of typecast

---

diff --git a/src/preload.c b/src/preload.c
index 8758d5c..2750b30 100644
--- a/src/preload.c
+++ b/src/preload.c
@@ -347,7 +347,7 @@ int bind(int socket, const struct sockaddr *addr, socklen_t addrlen)
 	int fd, ret;
 
 	if (fd_get(socket, &fd) == fd_rsocket) {
-		sin = (struct sockaddr_sin *) addr;
+		sin = (struct sockaddr_in *) addr;
 		if (!sin->sin_port || ntohs(sin->sin_port) > 1024)
 			return rbind(fd, addr, addrlen);
 
@@ -394,7 +394,7 @@ int connect(int socket, const struct sockaddr *addr, socklen_t addrlen)
 	int fd, ret;
 
 	if (fd_get(socket, &fd) == fd_rsocket) {
-		sin = (struct sockaddr_sin *) addr;
+		sin = (struct sockaddr_in *) addr;
 		if (ntohs(sin->sin_port) > 1024) {
 			ret = rconnect(fd, addr, addrlen);
 			if (!ret || errno == EINPROGRESS)
